queries.sql

```diff
@@ -1,3 +1,26 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Query 1 */
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+/* Query 2 */
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+/* Query 3 */
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+/* Query 4 */
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
```

Comment:
* Kindly add the screenshot with the results of your queries to your Pull Request description as stated in the [requirment](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=Remember%20to%20add%20the%20screenshot%20with%20the%20results%20of%20your%20queries%20to%20your%20Pull%20Request%20description.)